web/src/message_list_view.js

```diff
@@ -847,6 +860,7 @@ export class MessageListView {
 
         const new_message_groups = this.build_message_groups(message_containers, this.table_name);
         const message_actions = this.merge_message_groups(new_message_groups, where);
+
```

Comment:
Clean up your history here so that spaces like this don't get introduced.

Reply:
removed extra line